packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1932,32 +1932,16 @@ class GridPlacementData<DataType> extends ParentDataWidget<GridRenderObjectWidge
   }
 }
 
-/// A widget that uses the flex layout algorithm for its children.
+/// A widget that displays its children in a one-dimensional array.
 ///
 /// The [Flex] widget allows you to control the axis along which the children are
 /// placed (horizontal or vertical). This is referred to as the _main axis_. If
 /// you know the main axis in advance, then consider using a [Row] (if it's
-/// horizontal) or [Column] (if it's vertical) instead, since that will be less
+/// horizontal) or [Column] (if it's vertical) instead, because that will be less
```

Comment:
http://blog.apastyle.org/apastyle/2011/05/since-versus-because.html

I forget which of the style guides drilled it into me.